src/pages/MyNiwa/index.tsx

```diff
@@ -38,12 +38,12 @@ const useStyles = makeStyles((theme: Theme) =>
   })
 );
 
-export const MyGardens = () => {
+export const MyNiwa = () => {
   const classes = useStyles();
   const { userData } = useUserState();
   const [gardensApi, getUserGardens] = useApi(getGardens);
 
-  const myGardens = useMemo(() => gardensApi.response ?? [], [gardensApi]);
+  const myniwa = useMemo(() => gardensApi.response ?? [], [gardensApi]);
 
```

Comment:
I think for the state is ok to keep using garden terminology maybe `gardens` would be good